src/components/ui/menu-option/menu-option.scss

```diff
@@ -34,21 +28,25 @@ $menu-item-padding: 12px;
   padding: 0 $sidebar-padding 0 calc(#{$sidebar-padding} - 2px);
   user-select: none;
   white-space: nowrap;
+  border-bottom: solid var(--menu-option-border-color);
 
-  &.menu-option--focused,
-  &:focus,
   &:hover {
     background: var(--menu-option-bg-hover);
   }
 }
 
+.menu-option:last-of-type {
+  border-bottom: none;
+}
```

Comment:
I think you can put this in the style block above, same as it's done for `:hover`. So it would be this:

```css
&:last-of-type {
  border-bottom: none;
}
```

Reply:
Yes, my mistake. Thanks for notice